src/environments/environment.prod.ts

```diff
@@ -25,7 +25,7 @@ export const authConfig: AuthConfig = {
   issuer: '${AUTH_URL}/realms/backbase',
 
   // URL of the SPA to redirect the user to after login
-  redirectUri: '${PROTOCOL}//${HOSTNAME}:${PORT}${PATHNAME}${AUTH_LANDING_PAGE}',
+  redirectUri: window.location.href + 'transactions',
```

Comment:
this change will break docker implementation. What is the reason for that?

Reply:
Umm check the previous comment for the background. Lemme actually double-test this; you might be right

Reply:
this `${PROTOCOL}//${HOSTNAME}:${PORT}${PATHNAME}${AUTH_LANDING_PAGE}` are env variables coming from web-base:  https://github.com/Backbase/golden-sample-app/blob/master/docker-compose.yml#L13

Reply:
Also, the variable names we added to the golden sample don't seem to match with our original guideline:
https://backbase.atlassian.net/wiki/spaces/DE/pages/3020391409/How+to+dockerize+an+Angular+app

AUTH_LANDING_PAGE and AUTH_REDIRECT_PAGE don't need to be configurable per environment, as they are relative to the app, so we can remove those variables too.

Reply:
tested btw; it works